pages/index.js

```diff
@@ -16,29 +19,48 @@ export default function Home() {
 
       <Main>
         <h1>My App</h1>
-        <SearchbarLostItem />
+        <div>
+          <h4>Search1</h4>
```

Comment:
Your page has an `h1`, an `h2` and two `h4`. You should not skip headings for your HTML to be more semantic and also for accessibility reasons. More [info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).
If you want certain headings to appear visually in a different way, style them accordingly instead.